pvr.nextpvr/addon.xml.in

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.nextpvr"
-  version="22.1.0"
+  version="22.2.0"
   name="NextPVR PVR Client"
   provider-name="Graeme Blackley">
   <requires>@ADDON_DEPENDS@
-    <import addon="inputstream.ffmpegdirect" minversion="22.0.0"/>
+    <import addon="inputstream.ffmpegdirect" minversion="1.15.0" optional="true"/>
```

Comment:
This looks wrong. For Piers this would need to be starting with `22.`

Reply:
Historically I have left it like this because it means a don't have to update it with each minimum version of inputstream.ffmpegdirect,  even 22.0.0 will be wrong over the life of Piers, the bigger problem occurs when optional="true" is removed as happened with Piers.  Also the minimum does not give more versions to install from the addon, only API compatible versions are presented.  

The use case (client requested transcoding) is rare enough that I will be willing to support it if someone actually tries to install everything manually based only on addon.xml

Reply:
> even 22.0.0 will be wrong over the life of Piers

Not necessarily, only if the addon changes incompatible, which should only happen during Piers development phase not after Kodi 22.0 was officially released.

Reply:
Everything < 22.0.0 will not work with Piers. If you keep it like this you can also remove whatever you are doing with this addon in pvr.nextpvr. That would ahve the same effect. :-)

Reply:
Right but the build system looks at the repo and it shows

Reply:
minversion was very important in the early days off ffmpegdirect development. Now, not so much. 

Is your addon’s use of ffmpegdirect truly optional? I.e. you use the addon API to figure out what’s installed and only use ffmpegdirect if it’s available?

Reply:
> Is your addon’s use of ffmpegdirect truly optional? I.e. you use the addon API to figure out what’s installed and only use ffmpegdirect if it’s available?

Normally playback is direct play but there is an option to support server side transcoding and in that mode there is another option to support timeshifted transcoded playback with ffmpegdirect.  And yes there are are warnings if this use is attempted when it isn't installed or enabled.  https://github.com/kodi-pvr/pvr.nextpvr/blob/fec5c553ca5bc11d1dff22132824715b920577da/src/pvrclient-nextpvr.cpp#L276

Reply:
> Everything < 22.0.0 will not work with Piers. If you keep it like this you can also remove whatever you are doing with this addon in pvr.nextpvr. That would have the same effect. :-)

@ksooo I do not really understanding your comment since unsupported versions won't be in the repo https://mirrors.kodi.tv/addons/piers/addons.xml.gz  I don't think I have to remove any code.  

My point about 22.0.0 was simply that version doesn't exist so it is just as invalid as the lower version I was using, I think removing the minversion is the way to go.

Reply:
Let’s try removing minversion in nextpvr and see how it goes.